core/src/main/java/feign/ResponseInterceptor.java

```diff
@@ -0,0 +1,29 @@
+package feign;
```

Comment:
```suggestion
/*
 * Copyright 2012-2022 The Feign Authors
 *
 * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
 * in compliance with the License. You may obtain a copy of the License at
 *
 * http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under the License
 * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
 * or implied. See the License for the specific language governing permissions and limitations under
 * the License.
*/
package feign;
```
Add the license header. Tests will fail with "Some files do not have the expected license header" if it is not present.